config/passport.js

```diff
@@ -0,0 +1,45 @@
+const LocalStrategy = require("passport-local").Strategy;
```

Comment:
Have npm package imports first then local imports, make sure the imports are in alphabetical order in their respective groups.